### DISCUSSION TITLE
SSL issue

### DISCUSSION THREAD

**Discussion by dubeychandan996** [swan-io/swan-partner-frontend#657] [General]
dev server started

Banking: https://banking.swan.local:8080
Onboarding Individual: https://onboarding.swan.local:8080/onboarding/individual/start
Onboarding Company: https://onboarding.swan.local:8080/onboarding/company/start

above things works fine ..
now i want custom subdomain of every service and  when I try with reversepoxy of apach2 I am getting an error
 
Proxy Error
The proxy server could not handle the request
Reason: Error during SSL Handshake with remote server

**Comment by dubeychandan996** [comment]
in nginx:
Missing ALPN Protocol, expected `h2` to be available.
If this is a HTTP request: The server was not configured with the `allowHTTP1` option or a listener for the `unknownProtocol` event.

**Comment by bloodyowl** [comment]
hello,

I'm sorry but your issue doesn't include enough information.

does this happen locally or when you're trying to deploy? if locally, please follow the steps provided in the documentation: https://swan-io.github.io/swan-partner-frontend/setup
